code/modules/admin/verbs/playsound.dm

```diff
@@ -96,3 +96,29 @@ var/global/list/sounds_cache_local = list()
 		M.playsound_stop(CHANNEL_ADMIN)
 	log_admin("[key_name(src)] has stopped the global sound.")
 	message_admins("[key_name_admin(src)] has stopped the global sound.")
+
+/client/proc/set_bwoink_sound()
+	set category = "Fun"
+	set name = "Set Own Bwoink Sound"
+
+	if(!check_rights(R_ADMIN))
```

Comment:
I think you actually do need to also check for R_SOUND, so not all admins can do this somewhat adminboosive thingy